test/spirv-version-controls.ll

```diff
@@ -0,0 +1,33 @@
+; RUN: llvm-as %s -o %t.bc
+;
+; For now this is dummy tests to check that new options are consumed
+; RUN: llvm-spirv %t.bc -spirv-text --spirv-max-version=1.0 -o %t
```

Comment:
I have doubts that this test is really useful. I can expand it to test that resulting SPIR-V version will not be greater than specified by `--spirv-max-version`, but then it will look very similar to test for `DecorationMaxByteOffset`. Any other opinions?

Reply:
Probably not that useful then.  I'd expect that new tests for new SPIR-V features will make more use of the `--spirv-max-version` option anyway.

Reply:
Thanks, I will remove it then